stardogrest/api/interfaces.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+import (
+	"context"
+	"github.com/Azure/go-autorest/autorest"
+	"stardogrest"
```

Comment:
This needs to be `"github.com/vshn/stardog-userrole-operator/stardogrest"`, else compilation will fail.